mlos_bench/mlos_bench/tests/tunable_groups.py

```diff
@@ -0,0 +1,103 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""
+Common fixtures for mock TunableGroups.
+"""
+
+from typing import Any, Dict
+
+import pytest
+
+import json5 as json
+
+from mlos_bench.config.schemas import ConfigSchema
+from mlos_bench.tunables.covariant_group import CovariantTunableGroup
+from mlos_bench.tunables.tunable_groups import TunableGroups
+
+# pylint: disable=redefined-outer-name
+
+TUNABLE_GROUPS_JSON = """
```

Comment:
Don't we have it already in `conftest.py` on the same level? Maybe let's keep it there and reduce the duplication? (Also, I have changes coming to that JSON in my PR (adding weights, etc))

Reply:
In #644 I'm reusing these definitions for mlos-viz tests.  We did something similar in `mlos_bench/mlos_bench/tests/services/remote/ssh/fixtures.py`.

I can separate those changes out again if you want, but it was easiest to apply those patches all at once here.

Reply:
Actually, might make sense to move those to a `fixtures.py` as well.

Reply:
And actually, in my original splitting of the work I'd forgotten to completely cleanup the old `conftest.py`.  Should be fixed now.  Please have a look and let me know what you think.  Thanks!